act/fixtures/action.yml

```diff
@@ -0,0 +1,28 @@
+name: "CDS AI Code Review Action (act fixture)"
+description: "Perform code reviews and comment on diffs using OpenAI API."
+inputs:
+  GITHUB_TOKEN:
+    description: "GitHub token to interact with the repository."
+    required: true
+  OPENAI_API_KEY:
+    description: "OpenAI API key for GPT."
+    required: true
+  OPENAI_API_MODEL:
+    description: "OpenAI API model."
+    required: true
+  OPENAI_API_VERSION:
+    description: "OpenAI API version."
+    required: true
+  OPENAI_BASE_URL:
+    description: "Base URL to the OpenAI API model (OpenAI or Azure)."
+    required: true
+  exclude:
+    description: "Glob patterns to exclude files from the diff analysis"
+    required: false
+    default: ""
+runs:
+  using: "node16"
+  main: "dist/index.js"
+branding:
+  icon: "aperture"
+  color: "green"
```

Comment:
Not the real action, and suboptimal to store as a duplicate to the real action because of the way that `act` needs to be setup and its limitations.